CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 # Changes between v0.0.4-alpha.plus.rc7 and v0.0.4-alpha.plus.rc6:
 
+- version tag moved to page title (#1260)
+- fix opening a received link while on Send/Default fails to load Receive/Consent (#1238)
+- adjusted cancelation message (#1189)
```

Comment:
This is for the next release? So, we need another heading for "changes between rc7 and <next-version>, right and these three items should go into that, right?

Reply:
As these changes we deployed under alpha.plus.rc7 so thats why I thought to put under the same section. However as we can agreed on next version v0.1 I can transfer to seperate section